src/services/UserService.js

```diff
@@ -0,0 +1,12 @@
+import { api, getDefaultHeaders, manageError } from 'boot/axios';
+
+/**
+ * Load information about the current user.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object>} Promise with the logged-in user information on success
+ * otherwise an error.
+ */
+export async function load(sessionToken) {
```

Comment:
suggestion `loadUserInfo` / `loadCurrentUser` / same but `get` instead of `load`...

Reply:
see with you, updated.